lib/Types.cpp

```diff
@@ -214,42 +217,70 @@ Type *ExtractSmallerStructField(const DataLayout &DL, Type *Ty) {
   return Smaller;
 }
 
+// Returns a preferred type chosen from two types.
+// In particular:
+// - If either is nullptr, return the other.
+// - When one is an aggregate or vector whose first nested member at some
+//   level of nesting "matches" the other, then return the aggregate or vector.
+//   They both have the same machine byte address, but we prefer the larger
+//   type.  Here two types "match" if they're the same, or if they are
+//   scalar numeric types (integer or float0 with the same bit size.
```

Comment:
```suggestion
//   scalar numeric types (integer or float with the same bit size.
```

Reply:
I think that was meant to be a closing parenthesis.

```suggestion
//   scalar numeric types (integer or float) with the same bit size.
```